### ISSUE TITLE
Add support for passing attr() on Craft's included form controls

### ISSUE THREAD

**Issue by chasegiunta** [craftcms/cms#6696]
### Description

There's already support for `attr()` for the form input containers, which is nice.

For plugin developers that would like some extra control over inputs, while maintaining Craft's style choices for form controls (thus not rolling our own custom input), could we also pass `attr()` to the html input/control itself?

**Comment by brandonkelly** [comment]
`_includes/forms/text.html` and `_includes/forms/checkbox.html` already supported passing an `inputAttributes` variable, which can be set to a list of attributes just like what you pass into `attr()` or `|attr` (though I just realized that could have unintended consequences when passed to the `textField()` and `checkboxField()` macros, and fixed that in dec1e98f129ffb43cbd516a1bfc8784995ce92f5).

I’ve just added support for an `inputAttributes` variable in `_includes/forms/select.html` as well, so this will be possible as of the next release;

```twig
{% import '_includes/forms' as forms %}
{{ form.sselectField({
  inputAttributes: {
    ...
  }
]) }}
```

**Comment by brandonkelly** [comment]
Craft 3.5.6 is out now with that change.
